### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by T8mars** [rgthree/rgthree-comfy#334]
![image](https://github.com/user-attachments/assets/19fd4e69-9e3b-4954-81f8-8899adaeeb03)

all label show it

**Comment by T8mars** [comment]
![image](https://github.com/user-attachments/assets/f47de746-1cf4-4a52-98c4-c8290f4a0c0b)
json save normally,but when reopen the workflow,it stell show "标签",I ask friends ,they have same bug

**Comment by upboy-web** [comment]
我也遇到了同样的问题！！！！！！！！

**Comment by koilu** [comment]
+1

**Comment by winjvlee** [comment]
me too

**Comment by rgthree** [comment]
I believe this is an issue with another extension that is handling translations, as we've seen similar conflicts before and I cannot reproduce in a same-language setting.

**Comment by upboy-web** [comment]
意思是这是翻译节点问题吗？

**Comment by rgthree** [comment]
Yes, I believe a translation custom extension outside of rgthree-comfy is overwriting the label text with "标签"
